configure.ac

```diff
@@ -1,5 +1,5 @@
-AC_INIT(fleet-commander-admin, 0.7.6, aruiz@redhat.com)
-AC_COPYRIGHT([Copyright 2014,2015 Red Hat, Inc.])
+AC_INIT(fleet-commander-admin, 0.8.0, aruiz@redhat.com)
```

Comment:
It's better to use 0.7.99 instead of 0.8 as that would create clashes in the future with package names